src/Compilers/CSharp/Test/Emit3/RefStructInterfacesTests.cs

```diff
@@ -7112,6 +7112,130 @@ static void Main()
                 );
         }
 
+        [Fact]
+        public void Using_LanguageVersion_01()
+        {
+            var src1 = @"
+public ref struct S2 : System.IDisposable
+{
+    public void Dispose()
+    {
+    }
+}
+";
+
+            var comp1 = CreateCompilation(src1, targetFramework: s_targetFrameworkSupportingByRefLikeGenerics);
```

Comment:
Why aren't we checking langversion on implementation? Or is that forthcoming?

Reply:
> Why aren't we checking langversion on implementation? Or is that forthcoming?

Because this isn't a goal for this PR. I assume you are referring to a scenario covered in `ImplementAnInterface_01`.

Reply:
Yes, thanks for clarifying.